study-builder/tenants/prion/tenant.yaml

```diff
@@ -1,5 +1,6 @@
 tenant:
   default_directory: Username-Password-Authentication
+  picture_url: '##BASE_URL##/assets/images/project-logo-dark.svg'
```

Comment:
The password reset page has this:
```
    theme: {
        icon: "{{tenant.picture_url | default: '//cdn.auth0.com/styleguide/1.0.0/img/badge.png'}}",
            primaryColor: "{{tenant.colors.primary | default: '#ea5323'}}"
    }
```
I noticed on dev `tenant.picture_url` was pointing to some old `dev.prionregistry.org` URL. And on other environments, test/staging and even production does not have anything set for this. So I updated this to point to prion's project logo.